tempesta_fw/ss_skb.c

```diff
@@ -420,6 +420,9 @@ __split_linear_data(struct sk_buff *skb, char *pspt, int len, TfwStr *it)
 	__skb_fill_page_desc(skb, alloc, page, tail_off, tail_len);
 	get_page(page);
 
+	/* Prevent futher @skb->tail expanding */
+	__ss_skb_tail_lock(skb);
```

Comment:
Shouldn't we rather just put a check here `WARN_ON(__ss_skb_tail_is_locked(skb))`?